src/index.ts

```diff
@@ -227,7 +227,7 @@ export const getGPUTier = async ({
         closest = match;
       }
     }
-
+    if (!closest) return undefined;
```

Comment:
Could you add braces to the if statement, I usually do that to keep it more readable.